README.md

```diff
@@ -39,6 +39,10 @@ Include the CSS and JavaScript
 
 ## Installation
 
+To install GOV.UK Frontend NPM package in your application, need to create a [package.json](https://docs.npmjs.com/files/package.json) file in your application (if you haven't done so already) and add a package to it's devDependencies section.
```

Comment:
It'd need to be installed as a _non-dev_ dependency, right? You'd need Frontend to be included to actually use the application…

Reply:
true. updated